recipe/meta.yaml

```
@@ -1,26 +1,28 @@
-{% set version = "1.38.0" %}
+{% set version = "1.86.0" %}
 
 package:
   name: aws-sam-translator
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/a/aws-sam-translator/aws-sam-translator-{{ version }}.tar.gz
-  sha256: 0ecadda9cf5ab2318f57f1253181a2151e4c53cd35d21717a923c075a5a65cb6
+  sha256: a748dcd7886024cb7586abbbdbabe8c787c44c6547bb6602879d7bb8a6934d05
 
 build:
   noarch: python
   number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools
     - wheel
+    - pydantic ==2.5.3
   run:
-    - python >=3.6
+    - python
+    - pydantic ==2.5.3
```

```suggestion
    - pydantic >=1.8,<3
```
https://github.com/aws/serverless-application-model/blob/v1.86.0/requirements/base.txt#L6C9-L6C17